src/definitions/index.ts

```diff
@@ -0,0 +1,59 @@
+import length from '../definitions/length';
```

Comment:
Any technical reason for `import length from '../definitions/length';` rather than `import length from './length';`

Reply:
No technical reason at all. Copy pasta error that worked out in my favour 😄